src/plugins/kibana_react/public/no_data_page/no_data_card/elastic_agent_card.tsx

```diff
@@ -39,15 +40,10 @@ export const ElasticAgentCard: FunctionComponent<ElasticAgentCardProps> = ({
     typeof button !== 'string' && typeof button !== 'undefined' ? (
       button
     ) : (
-      <EuiButton
-        href={href}
-        onClick={cardRest?.onClick}
-        target={cardRest?.target}
-        fill={recommended}
-      >
+      <EuiButton href={href} onClick={cardRest?.onClick} target={cardRest?.target} fill>
         {button ||
           i18n.translate('kibana-react.noDataPage.elasticAgentCard.buttonLabel', {
-            defaultMessage: 'Find an integration for {solution}',
+            defaultMessage: 'Find a {solution} integration',
```

Comment:
@snide Quick question here, because of "language" 😏 this sentence strucutre will need to adapt to the Solution name like `Observability`. 

> Find a Observability integration

would actually need to be 

> Find **an** Observability integration

Any thoughts on how to do this automatically?

Reply:
Honestly. I struggled with it too. I think rather than using `solution`, we should probably just make a direct prop for `buttonText` or something.

Reply:
That's how we get inconsistent messaging... 🙁  Is it a hard no to go with the original sentence? It works for all solution names.

Reply:
I'll punt to you. It's not important enough to me and shouldn't block us. Feel free to revert that part. I can address later.

Reply:
I'm going to put back. If we really want to re-word, we'll need to do more detecting of the solution names.

Reply:
Also the `button` text **can** already be supplied/overridden. Just by passing a `string` as the `button` or completley change the `button` by passing a React Node.

https://github.com/elastic/kibana/blob/c13cfe3a0f81275a3a4df4a63c3013b54fa09c0d/src/plugins/kibana_react/public/no_data_page/no_data_page.tsx#L49-L52